src/clusterfuzz/_internal/platforms/android/util.py

```diff
@@ -37,3 +37,42 @@ def get_local_path(device_path):
   return os.path.join(
       root_directory,
       os.path.relpath(device_path, android.constants.DEVICE_FUZZING_DIR))
+
+
+def is_testcase_deprecated(platform_id=None):
+  """Whether or not the Android device is deprecated."""
+
+  # Platform ID for Android is of the form - |android:{codename}_{sanitizer}:{build_version}|
+  platform_id_fields = platform_id.split(':')
+  if len(platform_id_fields) != 3:
+    return False
+
+  codename_fields = platform_id_fields[1].split('_')
```

Comment:
Just for your consideration, I believe there are cases when no sanitizer is used. 

So the `.split('_')` method will be a no-operation. I do not think this is a problem.

Reply:
Yes correct. This is done to remove the sanitizer part (if present) so that the device part is obtained for comparison.